packages/helm-charts/attestation-service/templates/attestation.statefulset.yaml

```diff
@@ -55,7 +69,59 @@ spec:
         volumeMounts:
         - name: data
           mountPath: /root/.celo
+        - name: config
+          mountPath: /var/geth
+      - name: import-geth-account
+        image: {{ .Values.geth.image.repository }}:{{ .Values.geth.image.tag }}
+        imagePullPolicy: {{ .Values.imagePullPolicy }}
+        command: ["/bin/sh"]
+        args:
+        - "-c"
+        - |
+          geth account import --password /root/.celo/password /root/.celo/attestationSignerKey || true
+        volumeMounts:
+        - name: data
+          mountPath: /root/.celo
       containers:
+      - name: light-client
+        image: {{ .Values.geth.image.repository }}:{{ .Values.geth.image.tag }}
+        imagePullPolicy: IfNotPresent
+        command: ["/bin/sh"]
+        args:
+        - "-c"
+        - |-
+          set -euo pipefail
+          ATTESTATION_SIGNER_ADDRESS=`cat /root/.celo/attestationSignerAddress`
+          geth \
+            --password=/root/.celo/password \
+            --unlock=${ATTESTATION_SIGNER_ADDRESS} \
+            --ws \
+            --wsaddr 0.0.0.0 \
+            --wsorigins=* \
+            --wsapi=eth,net,web3,debug \
+            --networkid=${NETWORK_ID} \
+            --syncmode=light \
```

Comment:
Just confirming you want this to be light and not ultralight

Reply:
Honestly, was pretty arbitrary. I didn't really want the state growth of a full node so I just picked light. Not sure how reliable ultralight would be.